frontend/src/pages/ContributionView.jsx

```
@@ -0,0 +1,286 @@
+              <h4 className="sidebar-title">Overview of {course.title}</h4>
+              <div className="sidebar-items">
+                {course.content.map((item, index) => (
+                  <div 
+                    key={index} 
+                    className={`sidebar-item ${activeVideoIndex === index ? 'active' : ''}`}
+                    onClick={() => handleVideoSelect(index)}
+                  >
+                    <div className={`completion-indicator ${activeVideoIndex === index ? 'active' : ''} ${item.completed ? 'completed' : ''}`}>
+                      {activeVideoIndex === index && (
+                        <svg className="check-icon" viewBox="0 0 24 24">
+                          <path d="M5 13l4 4L19 7" />
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)